server/parameters/CoreConfig.java

```diff
@@ -364,7 +378,7 @@ else if (!outputs.get(outputName).isFile()) {
                     output = outputs.get(outputName).asFile();
                 }
 
-                public boolean isEnabled() {
```

Comment:
We have `enable` in the config entries, which has a natural usage on that level (e.g. `diagnostics.reporting.enable = true`), but we had multiple variations of `getter`s in the `Java` code as `enable` is a strange name for a property, and adjectives `isEnabled` and `enabled` are more suitable forms for it. I decided to just use `enabled` everywhere.